README.md

```diff
@@ -1,21 +1,23 @@
 # Setup & Configuration
 
-First of all clone this repo and install the dependencies:
+First clone this repo:
 
 ```sh
 git clone git@github.com:SharinPix/aws-image-backup.git
-cd aws-image-backup/
-npm install
 ```
 
-Once you have to configure appropriate environment variables.
+then install the dependencies:
+
+```sh
+npm install
+```
 
-**Note**: You can use the `.env` file for simplicity.
+Now you have to configure appropriate environment variables in the `serverless.yml` file.
```

Comment:
Now you have to configure appropriate *the* environment variables in the file `serverless.yml`.